src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -54,6 +60,37 @@ export default class BloodOfTheDragon extends Module {
 		this.addHook('complete', this._onComplete)
 	}
 
+	// duplicate code from other PRs
+	getActiveDrgBuffs() {
+		const active = []
+
+		if (this.combatants.selected.hasStatus(STATUSES.LANCE_CHARGE.id)) {
+			active.push(STATUSES.LANCE_CHARGE.id)
+		}
+
+		if (this.combatants.selected.hasStatus(STATUSES.BATTLE_LITANY.id)) {
+			active.push(STATUSES.BATTLE_LITANY.id)
+		}
+
+		if (this.combatants.selected.hasStatus(STATUSES.RIGHT_EYE.id) || this.combatants.selected.hasStatus(STATUSES.RIGHT_EYE_SOLO)) {
+			active.push(STATUSES.RIGHT_EYE.id)
+		}
+
+		return active
+	}
+
+	createTimelineButton(timestamp) {
+		return <Button
+			circular
+			compact
+			icon="time"
+			size="small"
+			onClick={() => this.timeline.show(timestamp - this.parser.fight.start_time,	timestamp - this.parser.fight.start_time)}
```

Comment:
Pretty unimportant but you've still got a few of those errant tabs running around.